### ISSUE TITLE
Can't detect/use/find iOS 15.4

### ISSUE THREAD

**Issue by mattleibow** [dotnet/xharness#838]
I just noticed that xharness can't seem to see simulators with iOS 15.4 for some reason.

When I do a test run, it says it can't find it nor can it find it in the sim search.

Maybe there is an issue, but my iOS 14.4 is fine. Also, there is no iOS 15.3 to find, not sure if that is relevant.

**Comment by premun** [comment]
I think, the problem might be with this file: https://github.com/dotnet/xharness/blob/main/src/Microsoft.DotNet.XHarness.iOS.Shared/SdkVersions.cs

This is something we should probably still improve but maybe we need to bump this file again. I can test that later

**Comment by premun** [comment]
@mattleibow what version of XHarness are you on? I just tried Xcode 13.3 + iOS 15.4 and everything worked fine:

```
XHarness command issued: apple test -t ios-simulator-64 -o o -a ../apps/ios-simulator-64/System.Numerics.Vectors.Tests.app
info: Preparing run for ios-simulator-64
info: Looking for available ios-simulator-64 simulators..
info: Found simulator device 'iPhone X (iOS 15.4) - created by XHarness'
info: Getting app bundle information from '/Users/premun/w/apps/ios-simulator-64/System.Numerics.Vectors.Tests.app'..
info: Uninstalling any previous instance of 'net.dot.System.Numerics.Vectors.Tests' from 'iPhone X (iOS 15.4) - created by XHarness'
info: Application 'net.dot.System.Numerics.Vectors.Tests' was uninstalled successfully
info: Installing application 'System.Numerics.Vectors.Tests' on 'iPhone X (iOS 15.4) - created by XHarness'
info: Application 'System.Numerics.Vectors.Tests' was installed successfully on 'iPhone X (iOS 15.4) - created by XHarness'
info: Starting test run for net.dot.System.Numerics.Vectors.Tests..
info: Application finished the test run successfully
info: Tests run: 1200 Passed: 1200 Inconclusive: 0 Failed: 0 Ignored: 0
info: Uninstalling the application 'net.dot.System.Numerics.Vectors.Tests' from 'iPhone X (iOS 15.4) - created by XHarness'
info: Application 'net.dot.System.Numerics.Vectors.Tests' was uninstalled successfully
XHarness exit code: 0
```

**Comment by mattleibow** [comment]
I think it fails if you set a version number on the device: `ios-simulator-64_15.4`

**Comment by premun** [comment]
Closing this as it got fixed here: #856
